### PR TITLE
Remove trueVerifier from main logic

### PR DESCRIPTION
**Pull request by corneliusweig**
This verifier implementation is only used in tests, so it should be defined there.

### CODE REVIEW

File: pkg/download/downloader_test.go [hunk 1]
```diff
@@ -444,3 +452,37 @@ func Test_extractArchive(t *testing.T) {
 		})
 	}
 }
+
+func TestTrueVerifier(t *testing.T) {
```
Comment (corneliusweig) [inline pkg/download/downloader_test.go]: This test was moved in order to co-locate it with the type definition.

Comment (ahmetb) [reply pkg/download/downloader_test.go]: Tbh this test doesn’t make sense. 
Now that trueVerifier is a test-only type, we don’t need a test for this anymore.

Comment (corneliusweig) [reply pkg/download/downloader_test.go]: Agreed :)

**Pull request merged**